starter_code/views/layout.hbs

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>ironbeers</title>
+
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css" integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS"
+    crossorigin="anonymous">
+  <link rel="stylesheet" href="/stylesheets/styles.css">
+</head>
+
+<body>
+  <ul class="nav">
+    <li class="nav-item">
+      <a class="nav-link active" href="/">Home</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" href="/beers">Beers</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" href="/random-beers">Random Beers</a>
+    </li>
+
+  </ul>
+
+
+  {{{body}}}
+
+
+  <footer>
+    <p class="p-home">Develop by Ironhack ;)</p>
```

Comment:
you could have directly used the footer tag to style this rather than adding a class